solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Features/Details/Scenes/LaunchDetails/Views/LaunchDetailsBodyView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LaunchDetailsBodyView.swift
+//  SpaceApp
+//
+//  Created by Razee Hussein-Jamal on 08/03/2022.
+//
+
+import SwiftUI
+
+struct LaunchDetailsBodyView: View {
+    let title: String
+    let description: String
+    let buttonTitle: String
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 10) {
+            Text(title)
+                .font(.title2)
+                .bold()
+                .frame(alignment: .leading)
+            
+            Text(description)
+                .lineLimit(3)
+
+            Button {
+                
+            } label: {
+                Text(buttonTitle)
+                    .bold()
+                    .frame(minWidth: 0, maxWidth: .infinity)
+            }.buttonStyle(SpaceButtonStyle())
+        }
+        .padding(15)
+    }
+}
+
+struct SpaceButtonStyle: ButtonStyle {
```

Comment:
**[CHANGE REQUEST]**
Please move this file to `Modules/Core/UI/Components/Buttons/Styles/`